Python_sem1/DZ1.py

```diff
@@ -0,0 +1,18 @@
+#Треугольник существует только тогда, когда сумма любых двух его сторон больше третьей.
+# Дано a, b, c - стороны предполагаемого треугольника. Требуется сравнить длину каждого отрезка-стороны с суммой двух других.
+# Если хотя бы в одном случае отрезок окажется больше суммы двух других, то треугольника с такими сторонами не существует.
+# Отдельно сообщить является ли треугольник разносторонним, равнобедренным или равносторонним.
+
+a = int(input("Введите длину стороны a : "))
+b = int(input("Введите длину стороны b : "))
+c = int(input("Введите длину стороны c : "))
+
+
+if a + b <= c or a + c <= b or b + c <= a:
+    print("Треугольник не существует")
+elif a != b and a != c and b != c:
```

Comment:
Можно проще, по аналогии с равносторонним
```suggestion
elif a != b != c:
```